<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Yeastar-PC" name="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes" tests="91" failures="12" timestamp="18 Oct 2018 08:09:17 GMT" time="5459.033" errors="0">
  <testcase name="M1_deleteOneRoute_no" time="19.942" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="B2_checkDID" time="71.320" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="B2_checkDID" time="72.404" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="J1_disTimeCondition" time="61.336" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="G8_timeDelay" time="485.311" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes">
    <failure type="java.lang.AssertionError" message="CDR被叫方检测 【预期：1105 [6200(1105)] 实际：6200】">
      <![CDATA[java.lang.AssertionError: CDR被叫方检测 【预期：1105 [6200(1105)] 实际：6200】
	at com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1092)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.failNotEquals(YsAssert.java:583)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEqualsImpl(YsAssert.java:146)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:120)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:203)
	at com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:289)
	at com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:278)
	at com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes.G8_timeDelay(InboundRoutes.java:1392)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </failure>
  </testcase> <!-- G8_timeDelay -->
  <testcase name="J2_deleteConference" time="18.518" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="G2_otherTime1" time="74.985" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="Q_recovery2" time="27.205" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="G5_addTime2_noHoliday" time="69.233" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="I8_timecondition_outbound_account" time="74.901" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes">
    <failure type="java.lang.AssertionError" message="CDR_Status检测 【预期：Answered 实际：No Answer】">
      <![CDATA[java.lang.AssertionError: CDR_Status检测 【预期：Answered 实际：No Answer】
	at com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1092)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.failNotEquals(YsAssert.java:583)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEqualsImpl(YsAssert.java:146)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:120)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:203)
	at com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:290)
	at com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:278)
	at com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes.I8_timecondition_outbound_account(InboundRoutes.java:1957)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </failure>
  </testcase> <!-- I8_timecondition_outbound_account -->
  <testcase name="O_deleteAllRoute" time="17.039" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="D3_ExtensionRange" time="105.137" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes">
    <failure type="java.lang.AssertionError" message="CDR_Status检测 【预期：Busy 实际：Voicemail】">
      <![CDATA[java.lang.AssertionError: CDR_Status检测 【预期：Busy 实际：Voicemail】
	at com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1092)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.failNotEquals(YsAssert.java:583)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEqualsImpl(YsAssert.java:146)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:120)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:203)
	at com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:290)
	at com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes.D3_ExtensionRange(InboundRoutes.java:735)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </failure>
  </testcase> <!-- D3_ExtensionRange -->
  <testcase name="B2_checkDID" time="71.445" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="H1_FeatureCode3" time="18.226" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="I8_timecondition_outbound_e1" time="0.042" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="H3_editDestination1" time="26.365" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="Q_recovery4" time="23.992" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="Q_recovery5" time="30.546" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="B2_checkDID" time="72.805" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="H1_FeatureCode1" time="30.000" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="D4_Voicemail" time="54.824" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="N_checkCallerIDPriority2" time="46.284" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="C2_checkPriority3" time="105.425" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="B1_checkTrunk" time="30.069" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes">
    <failure type="java.lang.AssertionError" message="CDR呼叫方检测 【预期：3001 [3001] 实际：callback3001】">
      <![CDATA[java.lang.AssertionError: CDR呼叫方检测 【预期：3001 [3001] 实际：callback3001】
	at com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1092)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.failNotEquals(YsAssert.java:583)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEqualsImpl(YsAssert.java:146)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:120)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:203)
	at com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:288)
	at com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:278)
	at com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Inbound.A_callfrom1_sip(Inbound.java:58)
	at com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes.B1_checkTrunk(InboundRoutes.java:127)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </failure>
  </testcase> <!-- B1_checkTrunk -->
  <testcase name="A2_init" time="42.598" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="E2_Outbound_iax" time="0.055" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="C3_recoveryRingGroup" time="61.558" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="H3_editDestination2" time="53.522" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="I7_timecondition_outbound_bri" time="0.058" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="B2_checkDID" time="73.342" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="E5_Outbound_e1" time="0.068" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="I5_timecondition_outbound_iax" time="0.022" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="K2_deletePartRoute_yes" time="15.529" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="A1_addExtensions" time="44.266" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="N_checkCallerIDPriority1" time="45.919" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="Q_recovery" time="93.632" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="G9_timeOrder3" time="124.930" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes">
    <failure type="java.lang.AssertionError" message="CDR被叫方检测 【预期：1105 [6200(1105)] 实际：6200】">
      <![CDATA[java.lang.AssertionError: CDR被叫方检测 【预期：1105 [6200(1105)] 实际：6200】
	at com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1092)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.failNotEquals(YsAssert.java:583)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEqualsImpl(YsAssert.java:146)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:120)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:203)
	at com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:289)
	at com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:278)
	at com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes.G9_timeOrder3(InboundRoutes.java:1481)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </failure>
  </testcase> <!-- G9_timeOrder3 -->
  <testcase name="L2_checkDIDPriority1" time="46.971" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="G5_addTime1_noWorkday" time="33.166" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="J3_checkConference" time="11.813" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="H1_FeatureCode2" time="39.857" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="B2_checkDID" time="90.320" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="A3_init" time="99.965" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="C2_checkPriority5" time="66.353" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="B2_checkDID" time="76.918" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="I6_timecondition_outbound_spx" time="0.069" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="L4_checkDIDPriority3" time="49.070" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="M2_deleteOneRoute_yes" time="28.065" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="H2_checkFeatureCode2" time="57.620" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="L1_import" time="16.954" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="E4_Outbound_bri" time="0.063" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="I9_timecondition_extensionRange" time="132.645" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="K3_deleteAllRoute_no" time="9.891" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="I4_timecondition_outbound_sip" time="61.981" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="M3_editRoute" time="56.689" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="G3_workday" time="72.641" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="I3_FxsExtension" time="25.099" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="B2_checkDID" time="71.220" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="E1_Outbound_sip" time="65.993" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="B2_checkDID" time="71.439" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="D1_Hangup" time="38.997" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="B3_checkCaller1" time="70.916" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes">
    <failure type="java.lang.AssertionError" message="CDR呼叫方检测 【预期：2005 [2X612] 实际：2001 [2001]】">
      <![CDATA[java.lang.AssertionError: CDR呼叫方检测 【预期：2005 [2X612] 实际：2001 [2001]】
	at com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1092)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.failNotEquals(YsAssert.java:583)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEqualsImpl(YsAssert.java:146)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:120)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:203)
	at com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:288)
	at com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:278)
	at com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes.B3_checkCaller1(InboundRoutes.java:245)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </failure>
  </testcase> <!-- B3_checkCaller1 -->
  <testcase name="G9_timeOrder2" time="106.138" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="G9_timeOrder1" time="71.554" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="G2_otherTime2" time="138.527" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes">
    <failure type="java.lang.AssertionError" message="CDR呼叫方检测 【预期：4001 [6100] 实际：2001 [2001]】">
      <![CDATA[java.lang.AssertionError: CDR呼叫方检测 【预期：4001 [6100] 实际：2001 [2001]】
	at com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1092)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.failNotEquals(YsAssert.java:583)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEqualsImpl(YsAssert.java:146)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:120)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:203)
	at com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:288)
	at com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:278)
	at com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes.G2_otherTime2(InboundRoutes.java:1102)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </failure>
  </testcase> <!-- G2_otherTime2 -->
  <testcase name="C2_checkPriority2" time="86.205" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="Q_recovery3" time="58.556" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="P2_checkSPX" time="0.039" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="P1_checkSPS" time="48.981" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="B6_editDIDAndCaller" time="28.454" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="G6_checkPriority_NoHoliday" time="32.003" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="K4_deleteAllRoute_yes" time="13.333" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="G4_holiday" time="78.030" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="C2_checkPriority4_deleteRoute" time="22.893" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="B2_checkDID" time="73.559" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="B4_checkCaller2" time="70.449" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes">
    <failure type="java.lang.AssertionError" message="CDR呼叫方检测 【预期：2005 [2X612] 实际：2001 [2001]】">
      <![CDATA[java.lang.AssertionError: CDR呼叫方检测 【预期：2005 [2X612] 实际：2001 [2001]】
	at com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1092)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.failNotEquals(YsAssert.java:583)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEqualsImpl(YsAssert.java:146)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:120)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:203)
	at com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:288)
	at com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:278)
	at com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes.B4_checkCaller2(InboundRoutes.java:285)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </failure>
  </testcase> <!-- B4_checkCaller2 -->
  <testcase name="C2_checkPriority1" time="67.812" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="D2_FxsExtensions" time="0.044" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="E6_Outbound_account" time="78.148" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes">
    <failure type="java.lang.AssertionError" message="CDR_Status检测 【预期：Answered 实际：No Answer】">
      <![CDATA[java.lang.AssertionError: CDR_Status检测 【预期：Answered 实际：No Answer】
	at com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1092)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.failNotEquals(YsAssert.java:583)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEqualsImpl(YsAssert.java:146)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:120)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:203)
	at com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:290)
	at com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:278)
	at com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes.E6_Outbound_account(InboundRoutes.java:1000)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </failure>
  </testcase> <!-- E6_Outbound_account -->
  <testcase name="I2_callBack" time="40.882" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes">
    <failure type="java.lang.AssertionError" message="预期2000会响铃 【预期：2 实际：4】">
      <![CDATA[java.lang.AssertionError: 预期2000会响铃 【预期：2 实际：4】
	at com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1092)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.failNotEquals(YsAssert.java:583)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEqualsImpl(YsAssert.java:146)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:120)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:429)
	at com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes.I2_callBack(InboundRoutes.java:1752)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </failure>
  </testcase> <!-- I2_callBack -->
  <testcase name="K1_deletePartRoute_no" time="49.053" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="J4_backup" time="43.487" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="H2_checkFeatureCode3" time="129.478" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes">
    <failure type="java.lang.AssertionError" message="CDR被叫方检测 【预期：1105 [6200(1105)] 实际：6200】">
      <![CDATA[java.lang.AssertionError: CDR被叫方检测 【预期：1105 [6200(1105)] 实际：6200】
	at com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1092)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.failNotEquals(YsAssert.java:583)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEqualsImpl(YsAssert.java:146)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:120)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:203)
	at com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:289)
	at com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:278)
	at com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes.H2_checkFeatureCode3(InboundRoutes.java:1670)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </failure>
  </testcase> <!-- H2_checkFeatureCode3 -->
  <testcase name="G1_addTimeCondition" time="101.806" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="H2_checkFeatureCode1" time="53.994" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="C1_addRoute" time="235.062" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="B5_checkDIDAndCaller" time="87.882" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes">
    <failure type="java.lang.AssertionError" message="CDR呼叫方检测 【预期：2005 [2X612] 实际：2001 [2001]】">
      <![CDATA[java.lang.AssertionError: CDR呼叫方检测 【预期：2005 [2X612] 实际：2001 [2001]】
	at com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1092)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.failNotEquals(YsAssert.java:583)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEqualsImpl(YsAssert.java:146)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:120)
	at com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:203)
	at com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:288)
	at com.yeastar.swebtest.driver.YSMethod.YS_Extension.checkCDR(YS_Extension.java:278)
	at com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes.B5_checkDIDAndCaller(InboundRoutes.java:326)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </failure>
  </testcase> <!-- B5_checkDIDAndCaller -->
  <testcase name="I1_Disa" time="53.770" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="L3_checkDIDPriority2" time="61.057" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="B2_checkDID" time="71.700" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
  <testcase name="G7_checkPriority_NoWorkday" time="98.569" classname="com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes"/>
</testsuite> <!-- com.yeastar.swebtest.testcase.DetailCase.pbxcase.InboundRoutes -->
